### DISCUSSION TITLE
Multiple dynamic placeholders in one component don't work unless they are named as column or row

### DISCUSSION THREAD

**Discussion by gitvadim** [Sitecore/jss#1764] [Help] [answered]
### Description

Looks like [this PR](https://github.com/Sitecore/jss/pull/1292/files) introduced a very 'interesting' change effectively preventing the Placeholder component from rendering multiple dynamic placeholders within one component unless they are named to match a very strict regex:
![image](https://github.com/Sitecore/jss/assets/8301895/5d977abe-0bfc-4605-bc4c-65c872858271)
What was the justification for this change? How do I go about implementing something similar to a Column or RowSplitter without naming my placeholders row or column? Not that this is a big deal, but I'm interested in knowing.

**Comment by ambrauer** [comment, marked as answer]
Hey @gitvadim This came up as an issue as well, and I believe should be resolved with [this PR](https://github.com/Sitecore/jss/pull/1745). This will go out with the next release (21.7.0, which is coming out shortly).
